src/Equinor.ProCoSys.IPO.WebApi/Controllers/Invitation/InvitationsController.cs

```diff
@@ -0,0 +1,60 @@
+﻿using System.ComponentModel.DataAnnotations;
+using System.Threading.Tasks;
+using Equinor.ProCoSys.IPO.Command.InvitationCommands.CreateInvitation;
+using Equinor.ProCoSys.IPO.Command.InvitationCommands.EditInvitation;
+using Equinor.ProCoSys.IPO.Query.GetInvitationById;
+using Equinor.ProCoSys.IPO.WebApi.Middleware;
+using MediatR;
+using Microsoft.AspNetCore.Mvc;
+using ServiceResult.ApiExtensions;
+
+namespace Equinor.ProCoSys.IPO.WebApi.Controllers.Invitation
+{
+    [ApiController]
+    [Route("Invitations")]
+    public class InvitationsController : ControllerBase
+    {
+        private readonly IMediator _mediator;
+
+        public InvitationsController(IMediator mediator) => _mediator = mediator;
+
+        [HttpGet("{id}")]
```

Comment:
Permissions? Should anyone be able to do this?

Reply:
This isn't ready yet. Added TODOs.